Automattic/newspack-theme
# Add Events template to theme, and single event sidebar option

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a copy of the Events Calendar template to the theme, so we can customize it easier.

It also adds an option to display the sidebar on the single events pages, to get it working closer to the "Default Page Template" option does for the old views (unfortunately this template doesn't work with the new v2 views). I did experiment with adding a second option for the events views (month, list, week, day, photos), but in almost all cases, there isn't really room, and things just end up feeling crowded. 

Though we do have a couple publishers displaying the sidebar with the old views, I think I'd like to hold off adding that to the new views, at least until we get a specific request for it. I'd love to have a second opinion on that, though, from anyone reviewing this PR! I do like to overthink things :) 

See #1433

### How to test the changes in this Pull Request:

1. Start with a test site with the Events Calendar set up, and a few events added. Make sure under your settings that:
     * AMP is turned off for these pages
     * That you're running the v2 views (under WP Admin > Events > Settings > Display, the Enable updated designs for all calendar views option should be checked).
     * That your site is set to use 'Tribe Events Styles' (under WP Admin > Events > Settings > Display)
     * That your site is set to use the 'Default Events Template' (under WP Admin > Events > Settings > Display).
2. Apply the PR and run `npm run build`
3. Navigate to Customizer > Events Calendar > Newspack Options and confirm you have an option to enable the sidebar on single events, and that it's off by default:

![image](https://user-images.githubusercontent.com/177561/127073453-bd0e4bba-b19c-457c-b306-e1038113df24.png)

4. Check the option, and click publish. 
5. Confirm that the regular theme sidebar is showing up on your single events, but not on other events pages:

![image](https://user-images.githubusercontent.com/177561/127073905-ff59651c-1397-46b7-95d0-2478df1be8b1.png)

6. Share your thoughts on this option, whether it makes sense or would be best removed for now, or expanded so it can be applied to all event views, regardless of space. 

### Other information:
